### PR TITLE
fix(reaction): fix issue when try to add emoji reaction

### PR DESCRIPTION
**Pull request by genie-magic**
**What this PR does** 📖
Fix 'If someone reacts to a message and you click to add that reaction as well nothing will happen'

**Which issue(s) this PR fixes** 🔨

AP-986

**Special notes for reviewers** 🗒️
Please try to react(emoji) to the message with User1 and after that try to add react to User1's emoji reaction.

### CODE REVIEW

**Comment by genie-magic** [comment]
image
For now we can add react to emoji which other user already reacted

**Review by phillsatellite — APPROVED** [review]

File: components/views/chat/message/reactions/Reactions.vue [hunk 1]
```diff
@@ -81,12 +81,16 @@ export default Vue.extend({
      * @example
      */
     quickReaction(emoji: String) {
-      this.$store.dispatch('ui/addReaction', {
+      const myTextilePublicKey = this.$TextileManager.getIdentityPublicKey()
+      this.$store.dispatch('textile/sendReactionMessage', {
```
Comment (josephmcg) [inline components/views/chat/message/reactions/Reactions.vue:85]: You can get your public key from store
`accounts.details.textilePubkey`

Comment (genie-magic) [reply components/views/chat/message/reactions/Reactions.vue]: Fixed.

Comment (genie-magic) [reply components/views/chat/message/reactions/Reactions.vue]: @josephmcg  can you review again?

**Review by josephmcg — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Comment by stavares843** [comment]
I noticed either the original user or the other user can remove the reaction, will add a ticket to be handled later

**Comment by stavares843** [comment]
[AP-1321](https://satellite-im.atlassian.net/browse/AP-1321)

**Pull request merged**